packages/athena/sodasql/dialects/athena_dialect.py

```diff
@@ -72,6 +74,23 @@ def create_connection(self):
             catalog_name=self.catalog)
         return conn
 
+    def __query_table(self, table_name):
+        query = f"""
+        SELECT *
+        FROM {table_name}
+        LIMIT 1
+        """
+        return query
+
+    def sql_test_connection(self) -> Union[Exception, bool]:
+        conn = self.create_connection()
```

Comment:
I think this should also be in try clause, because this can raise an exception

Reply:
create_connection method takes care of this already, no?

Reply:
All `create_connection` methods catch the connection exception: 

`except Exception as e:
            self.try_to_raise_soda_sql_exception(e)`

so if the exception is taken into account already